### DISCUSSION TITLE
Editing wikis in the Android app?

### DISCUSSION THREAD

**Discussion by VioletGiraffe** [community/community#41818] [Mobile]
### Select Topic Area

Product Feedback

### Body

I can't find a way to view, let alone - edit, wikis in the mobile app. I suspect this function is not implemented and I kindly ask for it to be added. But if it is already there, please point me to it.

**Comment by jfullstackdev** [comment]
use the browser version of GitHub, it function same as the GitHub Web

**Reply by VioletGiraffe** [reply]
What's the point of the app then? Everything the app does the web version also does.
